### PR TITLE
Marketing page

### PR DESCRIPTION
**Pull request by jasonramirez**
https://headquarters.io/tinysplash/stories/939

To Do
====

* Point `tinysplash.io` to our heroku app
* Create `/thank-you` page for FormKeep to go to after a submission
* Get a free FromKeep account

screen shot 2015-03-02 at 5 49 08 pm

### CODE REVIEW

File: app/views/pages/marketing.html.slim [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    div class="marketing-form"
+      = simple_form_for :guest,
+        url: "https://formkeep.com/f/6a792507ced8",
+        method: "post" do |form|
+        = form.input :email,
+          name: "email",
+          placeholder: "email",
+          wrapper: false,
+          label: false
+        = form.submit class: "button", value: "Sign Up"
```
Comment (Euraldius) [inline app/views/pages/marketing.html.slim]: Button title should use `i18n`

File: app/views/pages/marketing.html.slim [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    h5 = t("home.company_motto")
+    p class="margin-top"
+      = t("home.company_selling_point")
+
+    div class="marketing-form"
+      = simple_form_for :guest,
+        url: "https://formkeep.com/f/6a792507ced8",
+        method: "post" do |form|
+        = form.input :email,
+          name: "email",
```
Comment (Euraldius) [inline app/views/pages/marketing.html.slim]: `i18n`

File: app/views/pages/marketing.html.slim [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+          placeholder: "email",
```
Comment (Euraldius) [inline app/views/pages/marketing.html.slim]: i18n

**Comment by Euraldius** [comment]
How do we feel about adding some padding to the bottom? Having the image right up against the bottom of the page makes me think I could scroll down. I wonder if we can put a ©Tinysplash footer down there

**Comment by Euraldius** [comment]
Also the almost invisible placeholder in the email input form

File: app/views/pages/marketing.html.slim [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+section class="section section--hero section--full-screen hero--home"
+  div
+    = image_tag "tinysplash-logo.svg", alt: t("home.company_name")
+    h5 = t("home.company_motto")
+    p class="margin-top"
+      = t("home.company_selling_point")
+
+    div class="marketing-form"
+      = simple_form_for :guest,
+        url: "https://formkeep.com/f/6a792507ced8",
```
Comment (Euraldius) [inline app/views/pages/marketing.html.slim]: Let's toss this URL into an environment variable, so we aren't going to the real Formkeep account in dev

File: app/assets/stylesheets/views/marketing.scss [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    background: rgba($white, 0.5);
+    border: solid 2px transparent;
+    border-bottom-right-radius: 0;
+    border-top-right-radius: 0;
+    box-shadow: none;
+    display: inline-block;
+    padding: $base-button-padding $base-button-padding * 2;
+  }
+
+  input[type="submit"] {
```
Comment (Euraldius) [inline app/assets/stylesheets/views/marketing.scss]: @jasonramirez Is there a way to do this that will make Hound happy?

Comment (jasonramirez) [inline app/assets/stylesheets/views/marketing.scss]: Add a class to each input...

**Comment by jasonramirez** [comment]
> How do we feel about adding some padding to the bottom? Having the image right up against the bottom of the page makes me think I could scroll down. I wonder if we can put a ©Tinysplash footer down there. Or ©2015 or something. I should know how these © things work but I always forget

I agree with you. I'm also thinking that we could add a second section that you can scroll to that outlines the product in a little more detail. We are making a huge assumption that a product can be explained in 160 characters. It could be a great assumption, but I'm not entirely sure. Thoughts?

**Pull request merged**